engine/access/rest/routes/account_balance_test.go

```diff
@@ -0,0 +1,135 @@
+package routes
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	mocktestify "github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/access/mock"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// TestGetAccountBalance tests local getAccountBalance request.
+//
+// Runs the following tests:
+// 1. Get account balance by address at latest sealed block.
+// 2. Get account balance by address at latest finalized block.
+// 3. Get account balance by address at height.
+// 4. Get invalid account balance.
+func TestGetAccountBalance(t *testing.T) {
+	backend := &mock.API{}
```

Comment:
if you instantiate the mock like this, it will automatically call `AssertExpectations` for you
```suggestion
	backend := mock.NewAPI(t)
```